pkg/cloudprovider/providers/aws/aws_test.go

```diff
@@ -38,18 +38,18 @@ import (
 	kubeletapis "k8s.io/kubernetes/pkg/kubelet/apis"
 )
 
-const TestClusterId = "clusterid.test"
+const TestClusterID = "clusterid.test"
 const TestClusterName = "testCluster"
 
 type MockedFakeEC2 struct {
 	*FakeEC2Impl
 	mock.Mock
 }
 
-func (m *MockedFakeEC2) expectDescribeSecurityGroups(clusterId, groupName, clusterID string) {
+func (m *MockedFakeEC2) expectDescribeSecurityGroups(clusterID, groupName, _ string) {
```

Comment:
Can we just drop the third arg here?

Reply:
Yep, and the way it's being called we could really drop all the arguments. I didn't just to tread lightly but since this is just a test case I'll update it.

Reply:
Thank you - it does make it much easier to PRs when they don't get too big!  I'd be fine to see this go in as-is, or remove the underscore arg in this PR, but if you wanted to remove more args I think that would make more sense as a separate PR IMO.